src/core/config_mod.F90

```diff
@@ -1373,4 +1382,97 @@ SUBROUTINE Config_Process_DryDep( ConfigInput, Config, RC )
    END SUBROUTINE Config_Process_DryDep
 
 
+   !> \brief Process BVOC configuration
+   !!
+   !! This function processes the biogenic VOC configuration and performs the necessary actions based on the configuration.
+   !!
+   !! \param[in] ConfigInput The YAML configuration object
+   !! \param[inout] Config The configuration object
+   !! \param[out] RC The return code
+   !!
+   !! \ingroup core_modules
+   !!!>
+   SUBROUTINE Config_Process_Bvoc( ConfigInput, Config, RC )
+      USE CharPak_Mod,    ONLY : StrSplit
+      USE Error_Mod
+      USE Config_Opt_Mod,  ONLY : ConfigType
+
+      TYPE(QFYAML_t),      INTENT(INOUT) ::ConfigInput      ! YAML Config object
+      TYPE(ConfigType),     INTENT(INOUT) :: Config   ! Input options
+
+      !
+      ! !OUTPUT PARAMETERS:
+      !
+      INTEGER,        INTENT(OUT)   :: RC          ! Success or failure
+      ! !LOCAL VARIABLES:
+      !
+      ! Scalars
+      LOGICAL                      :: v_bool
+      INTEGER                      :: v_int
+      ! Reals
+      REAL(fp)                     :: v_real
+      ! Strings
+      CHARACTER(LEN=255)           :: thisLoc
+      CHARACTER(LEN=512)           :: errMsg
+      CHARACTER(LEN=QFYAML_StrLen) :: key
+
+      !========================================================================
+      ! Config_Process_Bvoc begins here!
+      !========================================================================
+
+      ! Initialize
+      RC      = CC_SUCCESS
+      thisLoc = ' -> at Config_Process_Bvoc (in CATChem/src/core/config_mod.F90)'
+      errMsg = ''
+      ! TODO #105 Fix reading of config file
+      key   = "process%bvoc%activate"
+      v_bool = MISSING_BOOL
+      CALL QFYAML_Add_Get( ConfigInput, TRIM( key ), v_bool, "", RC )
+      IF ( RC /= CC_SUCCESS ) THEN
+         errMsg = 'Error parsing ' // TRIM( key ) // '!'
+         CALL CC_Error( errMsg, RC, thisLoc )
+         RETURN
+      ENDIF
+      Config%bvoc_activate = v_bool
+
+      key   = "process%bvoc%scheme_opt"
+      v_int = MISSING_INT
+      CALL QFYAML_Add_Get( ConfigInput, TRIM( key ), v_int, "", RC )
+      IF ( RC /= CC_SUCCESS ) THEN
+         errMsg = TRIM( key ) // 'Not Found, Setting Default to 1'
+         CALL CC_Error( errMsg, RC, thisLoc )
+         v_int = 1 ! default is one
```

Comment:
I think if you want to allow using this default, you should use `CC_Warning` instead here (and the similar places below), since the `rc` is checked in `Read_Input_File` and it looks like it would cause early exit. Did you try it?